lib/runtime2/threepio/trace/TraceManager.ts

```diff
@@ -0,0 +1,148 @@
+import util from 'util';
+import { TimePoint, TimePointOptions } from './TimePoint';
+import { TaskFlowSpan, SpanContext } from './TaskFlowSpan';
+
+interface CallNode {
+  requestId: string;
+  TaskFlowSpan: TaskFlowSpan[];
+  children: CallNode[];
+}
+export class TraceManager {
+  #callGragh: CallNode[] = [];
```

Comment:
```suggestion
  #callGraghs: CallNode[] = [];
```
这里一个 callGraph 代表的是一次 call 吧？